All Alpine319 legs are failed to build CoreCLR component

`main` build: https://dev.azure.com/dnceng/internal/_build/results?buildId=2434119

Working build: https://dev.azure.com/dnceng/internal/_build/results?buildId=2433339&view=logs&s=9b073b6c-199e-5782-2adb-83c30ea48e21&j=b9abe003-a5da-5729-0444-865506f15e35

Error:

      /vmr/src/runtime/src/coreclr/debug/daccess/cdac.cpp:51:25: error: assigning to 'intptr_t' (aka 'long') from incompatible type 'std::nullptr_t'
         51 |         m_cdac_handle = NULL;
            |                         ^~~~
      /usr/include/time.h:11:14: note: expanded from macro 'NULL'
         11 | #define NULL nullptr
            |              ^~~~~~~
      /vmr/src/runtime/src/coreclr/debug/daccess/cdac.cpp:65:23: error: invalid operands to binary expression ('intptr_t' (aka 'long') and 'std::nullptr_t')
         65 |     if (m_cdac_handle != NULL)
            |         ~~~~~~~~~~~~~ ^  ~~~~
      2 errors generated.

--- a/src/coreclr/debug/daccess/cdac.cpp
+++ b/src/coreclr/debug/daccess/cdac.cpp
@@ -48,7 +48,7 @@ CDAC::CDAC(HMODULE module, uint64_t descriptorAddr, ICorDebugDataTarget* target)
 {
     if (m_module == NULL)
     {
-        m_cdac_handle = NULL;
+        m_cdac_handle = 0;
         return;
     }
 
@@ -62,7 +62,7 @@ CDAC::CDAC(HMODULE module, uint64_t descriptorAddr, ICorDebugDataTarget* target)
 
 CDAC::~CDAC()
 {
-    if (m_cdac_handle != NULL)
+    if (m_cdac_handle)
     {
         decltype(&cdac_reader_free) free = reinterpret_cast<decltype(&cdac_reader_free)>(::GetProcAddress(m_module, "cdac_reader_free"));
         _ASSERTE(free != nullptr);

